include/nabla.h

```diff
@@ -18,7 +18,7 @@
 #define __NABLA_H_INCLUDED__
 
 // meta info
-#include "nbl/git_info.h"
+#include "git_info.h"
```

Comment:
can you show me what's in this header after generation

Reply:
git_info.cpp after generation
```cpp
#include "git_info.h"

namespace nbl{
    const GitInfo git_info = {
    .isPopulated = true,
    .hasUncommittedChanges = true,
    .commitAuthorName =  "tad1",
    .commitAuthorEmail = "tadziobrzeski@gmail.com",
    .commitHash = "84ff6ebd8a2db2f88de01e5bc7d142ebb205aa7d",
    .commitShortHash = "84ff6ebd8",
    .commitDate = "2024-09-09 19:18:37 +0200",
    .commitSubject = "use NBL_ROOT_PATH as git wd",
    .commitBody = "",
    .describe = "shared_build_msvc_assemblies-3959-g84ff6ebd8",
    .branchName = "tad_add_git_commit_hash",
    .latestTag = "v0.4-delta-2681-g84ff6ebd8",
    .latestTagName = "v0.4-delta"
    };

    const GitInfo getGitInfo() {
        return git_info;
    };
} // namespace nbl
```

internal useage
```cpp
#include "git_info.h" //or #include "nabla.h" 
// ....
m_logger.log("last commit %s:%s", system::ILogger::ELL_ERROR, nbl::git_info.commitShortHash, nbl::git_info.commitSubject);
// ....
```

exterrnal `.dll` useage
```cpp
#include "nabla.h" 
// ....
m_logger->log("last commit %s:%s", ILogger::ELL_ERROR, nbl::getGitInfo().commitShortHash, nbl::getGitInfo().commitSubject);
// ....
```

Reply:
looks good